scripts/automate.py

```diff
@@ -1014,7 +1014,7 @@ def pluralize(count: int, singular: str, plural: Optional[str]=None):
 
     subete.repo.logger.setLevel(logging.WARNING)  # Reduce the noise of subete
     log.info("Loading repos (this may take several minutes)")
-    repo = subete.load()
+    repo = subete.load(sample_programs_website_repo_dir=".")
```

Comment:
I assume this is much faster because the repo is already cloned when this script is executed.

Reply:
Yes, it is faster, but the primary reason was that I couldn't see what the new project article looked like since it was cloning the already committed repo, and that project didn't exist yet.